pkg/network/tracer/tracer.go

```diff
@@ -451,6 +453,18 @@ func (t *Tracer) getRuntimeCompilationTelemetry() map[string]network.RuntimeComp
 			tm.RuntimeCompilationDuration = duration
 		}
 		result[assetName] = tm
+
+		statsdMetricPrefix := "datadog.system_probe." + assetName
```

Comment:
I think we should have the asset name be a tag, rather than part of the metric name. WDYT?

Reply:
Works for me!